source/Cute/Commands/Server/ServerSechedulerCommand.cs

```diff
@@ -1,16 +1,231 @@
+            : ContentfulConnection.GetAllPreviewEntries<CuteContentSyncApi>().OrderBy(e => e.Order).ToArray();
+
+        if (!cronTasks.Any())
+        {
+            throw new CliException($"No data sync entries found.");
+        }
+
+        _cronTasks = cronTasks.ToDictionary(t => Guid.NewGuid(), t => t);
+    }
+
+    private void RefreshScheduler()
+    {
+        if (_settings is null) return;
+
+        var settings = _settings;
+
+        _scheduler.Stop();
+
+        _scheduler.RemoveAllTasks();
+
+        foreach (var cronTask in _cronTasks)
+        {
+            var syncApiKey = cronTask.Value.Key;
+
+            if (syncApiKey is null) continue;
+
+            var frequency = cronTask.Value.Schedule;
+
+            if (frequency is null) continue;
+
+            var cronSchedule = frequency.ToCronExpression().ToString();
+
+            _taskRunningStates[cronTask.Key] = false;
+
+            var scheduledTask = new AsyncScheduledTask(
+                cronTask.Key,
+                CrontabSchedule.Parse(cronSchedule),
+                ct =>
+                {
+                    if (_taskRunningStates[cronTask.Key])
+                    {
+                        return Task.CompletedTask;
+                    }
+
+                    // Set the running state to true
+                    _taskRunningStates[cronTask.Key] = true;
+
+                    try
+                    {
+                        return Task.Run(() => ProcessContentSyncApyAndDisplaySchedule(cronTask.Value));
+                    }
+                    finally
+                    {
+                        // Reset the running state
+                        _taskRunningStates[cronTask.Key] = false;
+                    }
+                }
+            );
+
+            _scheduler.AddTask(scheduledTask);
+        }
+
+        _scheduler.Start();
+    }
+
+    private void RefreshSchedule([FromForm] string command, HttpContext context)
+    {
+        if (!command.Equals("reload")) return;
+
+        LoadSyncApiEntries();
+
+        RefreshScheduler();
+
+        context.Response.Redirect("/");
+    }
+
+    private async Task ProcessContentSyncApyAndDisplaySchedule(CuteContentSyncApi entry)
+    {
+        await ProcessContentSyncApi(entry);
+
+        DisplaySchedule();
+    }
+
+    private void DisplaySchedule()
+    {
+        foreach (var entry in _cronTasks.Values)
+        {
+            if (entry.Key is null) continue;
+
+            var frequency = entry.Schedule;
+
+            var cronSchedule = frequency?.ToCronExpression().ToString();
+
+            _console.WriteNormal("Content sync-api '{syncApiKey}' usually scheduled to run on schedule '{frequency} ({cronSchedule})'",
+                entry.Key, frequency, cronSchedule);
+        }
+    }
+
+    private async Task ProcessContentSyncApi(CuteContentSyncApi cuteContentSyncApi)
+    {
+        string[] args = ["content", "sync-api", "-k", cuteContentSyncApi.Key, "-a", "--force"];
```

Comment:
use full command switches in code eg. --apply etc so that the intent is clear without having to look up switches in code